gen/go/proto/v1/config.proto

```diff
@@ -18,8 +18,8 @@ message AgentConfig {
   Javaagent javaagent = 7;
   LogConfig logging = 8;
   MetricsConfig metrics_config = 9;
-  // represents the environment name of agent
-  google.protobuf.StringValue environment = 10;
+  // AgentAttributesConfig has the parameters specific to an agent
+  AgentAttributesConfig agent_attributes_config = 10;
```

Comment:
This also seems like the agentAttributes processor config we have. 
Need to think about another name maybe?

Reply:
+1

Reply:
Hmm..thought about it while naming.  Want to have some thing which specifies this corresponds of parameters specific to an agent (as we may populate it in future). Something like `AgentParametersConfig`?. I am out of naming ideas tbh.

Reply:
How about HostDetails or ServixeDetails ?

Reply:
Should I go with `ServiceDetails` as @varkey98 suggested? Or do you have any suggestions @ryanericson @tim-mwangi @shashank11p.

Reply:
AgentHostAttributes is also a potential name, I like this better